dist/js/stacks.js

```diff
@@ -1757,7 +1757,7 @@ Copyright © 2019 Basecamp, LLC
 ;
 
 /**
- * @popperjs/core v2.4.0 - MIT License
+ * @popperjs/core v2.4.4 - MIT License
```

Comment:
🤔 This probably shouldn't be bumped in _this_ PR.

Reply:
Interesting that happened automatically

Reply:
Hey, no judgement. I bet `npm install` trolled you here.